manifest.json

```diff
@@ -3,23 +3,20 @@
   "applications": {
     "gecko": {
       "id": "copypatch@kiszka.org",
-      "strict_min_version": "78.4.0",
-      "strict_max_version": "102.*"
+      "strict_min_version": "102.0"
```

Comment:
Must become 106.0 due to using the tab parameter of the onCommand callback.

Reply:
It was backported to 102:
https://searchfox.org/comm-esr102/source/mail/components/extensions/parent/ext-commands.js#46-49

You sure it does not work?

Reply:
It is documented in the 102 branch in of the API as well:
https://webextension-api.thunderbird.net/en/102/commands.html#events

Reply:
I was reading https://webextension-api.thunderbird.net/en/latest/commands.html - seems your documentation is outdated then. To which version of 102 was this added then exactly?

Reply:
Ah, 102.3.3 then.

Reply:
But to be precise, it must be set to 102.4.0. So indeed a good catch! (102.3.3 does not exist in the list of [allowed versions](https://addons.thunderbird.net/pages/appversions/))